packages/desktop-client/src/components/autocomplete/PayeeAutocomplete.tsx

```diff
@@ -288,6 +288,7 @@ export function PayeeAutocomplete({
       focused={payeeFieldFocused}
       inputProps={{
         ...inputProps,
+        autoCapitalize: 'words',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I believe the previous lowercase variation was more correct here (according to the spec of this attribute)
```suggestion
        autocapitalize: 'words',
```

Reply:
Will take a look at this later after work, just went with what the autosuggest was wanting so I've got no problems with updating that

Reply:
@MatissJanis the suggestion does not appear to be valid since the type is coming from [here](https://use-form.netlify.app/interfaces/_node_modules__types_react_index_d_.react.inputhtmlattributes#autocapitalize)
![image](https://github.com/user-attachments/assets/d2696da1-aee1-48cf-bfae-1b097b1e20c5)

Reply:
Thanks for checking! 🙇